app.py

```diff
@@ -0,0 +1,138 @@
+import base64
+import json
+import logging
+import os
+import sys
+import boto3
+
+from chalice import Chalice
+
+app_name = 'CMSFilterApp'
+
+app = Chalice(app_name=app_name)
+
+# Initialize the logger
+logging.basicConfig(filename='app.log', filemode='w', format='%(name)s - %(levelname)s - %(message)s')
+logger = logging.getLogger(app_name)
+
+# Set the log level from Environment variable LOG_LEVEL
+if os.environ.get('LOG_LEVEL') is None:
+    logger.setLevel(logging.DEBUG)
+else:
+    logger.setLevel(os.environ['LOG_LEVEL'])
```

Comment:
this can be improved by using the vact that os.environment.get can take a default value. e.g.

logger.setLevel(os.environ.get('LOG_LEVEL', logging.DEBUG).

Also, not sure if DEBUG should be the default level. the default level should production friendly. Let's use INFO instead.